modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusManager.scala

```diff
@@ -177,6 +190,28 @@ object ConsensusManager {
             .handleErrorWith(logger.error(_)(s"Error checking for consensus state update {key=${key.show}}"))
         }.void
 
+      private def getPeerCandidates(peers: List[Peer]): F[List[Peer]] = {
+        case class PeerSnapshot(ordinal: SnapshotOrdinal, hash: Hash, peer: Peer)
```

Comment:
It can be a named tuple. You don't use it outside of this fn anyway

Reply:
Please elucidate. Aren't named tuples in Scala case classes?

Reply:
```scala
type PeerSnapshot = (SnapshotOrdinal, Hash)
```

you create a case class to convert it to tuple anyway

Reply:
@cngo-github that's why we use pattern matching, for example:
![image](https://user-images.githubusercontent.com/4959931/231790193-d64195e3-2a2b-4e6e-b6ed-b4497add6255.png)